test/cli/run/run-eval.test.ts

```diff
@@ -196,3 +217,18 @@ describe("echo | bun run -", () => {
 
   group(run);
 });
+
+test("process._eval (undefined for normal run)", async () => {
+  const cwd = tmpdirSync();
+  const file = join(cwd, "test.js");
+  writeFileSync(file, "console.log(process._eval)");
```

Comment:
```suggestion
  writeFileSync(file, "console.log(typeof process._eval)");
```

let's make sure it's not the string "undefined"